indra/llui/lltexteditor.h

```diff
@@ -313,10 +313,9 @@ class LLTextEditor :
     template <typename STRINGTYPE>
     void            pasteTextWithLinebreaks(const STRINGTYPE& clean_string)
     {
-        pasteTextWithLinebreaks<LLWString>(ll_convert(clean_string));
+        pasteTextWithLinebreaksImpl(ll_convert(clean_string));
```

Comment:
What about
```
    void            pasteTextWithLinebreaks(const STRINGTYPE& clean_string)
    {
        if constexpr (std::is_same_v<LLWString, STRINGTYPE>())
            pasteTextWithLinebreaksImpl(clean_string);
        else
            pasteTextWithLinebreaks<LLWString>(ll_convert(clean_string));
    }
```
to prevent an unnecessary ll_convert if not needed?